packages/sdk/src/tests/controllers/SubscriberContoller.test.ts

```diff
@@ -93,15 +97,15 @@ describe('SubscriberController', () => {
         expect(mockEditorApi.onScrubberPositionChanged).toHaveBeenCalledTimes(1);
         expect(mockEditorApi.onScrubberPositionChanged).toHaveBeenLastCalledWith('test');
     });
-    it('Should be possible to subscribe to onSelectedFrameLayoutChanged', async () => {
-        await mockedSubscriberController.onSelectedFrameLayoutChanged('2');
-        expect(mockEditorApi.onSelectedFrameLayoutChanged).toHaveBeenCalledTimes(1);
-        expect(mockEditorApi.onSelectedFrameLayoutChanged).toHaveBeenCalledWith(2);
+    it('Should be possible to subscribe to onSelectedFramesLayoutChanged', async () => {
```

Comment:
Re-add the tests for the deprecated methods to ensure they produce the intended results